java-frontend/src/main/java/org/sonar/java/JavaIssue.java

```diff
@@ -81,14 +81,15 @@ public JavaIssue addSecondaryLocation(InputFile file, int startLine, int startLi
   }
 
   public JavaIssue addFlow(InputFile file, List<List<AnalyzerMessage>> flows) {
-    for (List<AnalyzerMessage> flow : flows) {
+    // FIXME SONARJAVA-2111 all flows should be reported
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)